apps/foundation-demo/src/app/checkout/components/shipping/shipping-options/shipping-options.component.html

```diff
@@ -1,7 +1,7 @@
 <fieldset class="shipping-options">
   <legend class="shipping-options__title">Shipping Options</legend>
-  <div class="shipping-options__radio" *ngFor="let option of shippingOptions" (click)="onSelectShippingOption(option.id)">
-    <input type="radio" id="{{option.id}}" name="shipping-options" [checked]="selectedShippingOption == option.id"/>
-    <label for="{{option.id}}">{{option.text}}</label>
+  <div class="shipping-options__radio" *ngFor="let option of shippingOptions; let i = index" (click)="onSelectShippingOption(i)">
```

Comment:
While the idea here: O(1) lookup time makes sense for perf, we probably can't get away with it and it will likely cause further operations down the pipeline. I think we'll have to stick with `id` instead of `index`

Reply:
done